motioneye/config.py

```diff
@@ -1690,7 +1703,19 @@ def motion_camera_dict_to_ui(data):
 
     command_end_notifications = []
     for e in on_event_end:
-        if e.count('relayevent') or e.count('eventrelay.py'):
+        if ' webhook ' in e:
+            e = shlex.split(
+                utils.make_str(e)
+            )  # poor shlex can't deal with unicode properly
```

Comment:
Move the comment to its own line to avoid the three-line wrap...
```suggestion
            # poor shlex can't deal with unicode properly
            e = shlex.split(utils.make_str(e))
```
My bet is that Python 3 code has less trouble with Unicode because all `str` are Unicode utf-8 by default.

Reply:
I don't even know yet what `shlex` actually is and this is used for 😄. But your suggestion makes sense. #2351 added a few more such cases.

Reply:
https://docs.python.org/3/library/shlex.html

Reply:
You mean the following would work as well?
```py
e = shlex.split(e)
```

Reply:
If `e = Elephant()` or e = `b`bytes`` then no, it won't work but `dev` has no `utils`.
https://github.com/motioneye-project/motioneye/blob/dev/motioneye/utils.py

Reply:
Works well indeed without `utils.make_str` with various web hooks and commands tested.